data/102/064/247/102064247.geojson

```diff
@@ -41,7 +41,10 @@
         404227445
     ],
     "wof:breaches":[],
-    "wof:concordances":{},
+    "wof:concordances":{
+        "fra-insee:code":"1301"
```

Comment:
Just interesting, as this place has no name.. was this value found through an existing property? Or a spatial/data join?